### PR TITLE
Operator README

### PR DESCRIPTION
**Pull request by smcclem**
I started some basic updates, but I will need help filling in more detail.  

Here is the matching CASE bundle README. 

https://github.ibm.com/websphere/liberty-case-bundle/blob/main/stable/ibm-websphere-liberty-bundle/case/ibm-websphere-liberty/README.md

### CODE REVIEW

**Comment by leochr** [comment]
@smcclem some content with the relevant short URLs for WebSphere Liberty operator. Let's create this for now. Since this is not a functional change, we can modify these later once ID team makes progress with the documentation.

```md
## Prerequisites

Review the [system requirements](https://ibm.biz/wlo-sys-req) for details. 

## Supported platforms

Red Hat OpenShift Container Platform 4.8 or newer installed on one of the following platforms:
- Linux x86_64

## Documentation

See [IBM WebSphere Automation documentation](https://ibm.biz/wlo-docs).

## SecurityContextConstraints Requirements

The IBM WebSphere Liberty operator runs in the `restricted` security context constraints.

## Resources Required

Review the [resource requirements](https://ibm.biz/wlo-reqs) before you plan to install IBM WebSphere Liberty operator.

## Installing

Install the IBM WebSphere Liberty operator to the desired namespace and create an instance of the [WebSphereLibertyApplication custom resource](https://ibm.biz/wlo-crs).

## Configuration

The WebSphere Liberty operator provides various customization options to configure the application deployments. Please see [custom resource](https://ibm.biz/wlo-crs) for details.

## Storage

Please see the [storage requirements](https://ibm.biz/wlo-reqs) for details.

## Limitations

IBM WebSphere Automation is not available on Power or Z architectures. Please see the [limitations](https://ibm.biz/wlo-limits) for additional information.

```

File: config/manifests/description.md [hunk 1]
```diff
@@ -1,44 +1,89 @@
...
+Install the IBM WebSphere Liberty operator to the desired namespace and create an instance of the [WebSphereLibertyApplication custom resource](https://ibm.biz/wlo-crs).
+
+## Configuration
+
+The WebSphere Liberty operator provides various customization options to configure the application deployments. Please see [custom resource](https://ibm.biz/wlo-crs) for details.
+
+## Storage
+
+Please see the [storage requirements](https://ibm.biz/wlo-reqs) for details.
+
```
[the remark this answers is not in the document]
Comment (smcclem) [reply config/manifests/description.md:47]: @leochr   Is it the same limitations as automation?

Limitations
IBM WebSphere Automation is not available on IBM Power® Systems or IBM z Systems® architectures. Please see the [limitations](https://ibm.biz/WebSphereAutoLimits) for additional information.

[the remark this answers is not in the document]
Comment (leochr) [reply config/manifests/description.md:47]: @smcclem Yes, WL operator has the same limitation. So we can add:

```markup
## Limitations

IBM WebSphere Liberty Operator is not available on Power or Z architectures. Please see the [limitations](https://ibm.biz/wlo-limits) for additional limitations.
```

File: bundle/manifests/ibm-websphere-liberty.clusterserviceversion.yaml [hunk 2]
```diff
@@ -360,51 +360,91 @@ spec:
...
+    
+    ### Integration with OpenShift Serverless
+    Deploy your serverless runtime component using a single toggle.  The Operator will convert all of its generated resources into [Knative](https://knative.dev) resources, allowing your pod to automatically scale to 0 when it is idle.
+    
+    ### Integration with OpenShift's Topology UI
+    We set the corresponding labels to support OpenShift's Developer Topology UI, which allows you to visualize your entire set of deployments and how they are connected.  icon:
```
Comment (leochr) [inline bundle/manifests/ibm-websphere-liberty.clusterserviceversion.yaml:447]: Format is not right. The `icon` field should be on a new line.

**Comment by smcclem** [comment]
@leochr  I think this is all updated with the code review comments now.

**Review by leochr — APPROVED** [review]
@smcclem Thanks for the updates.

**Pull request merged**